sdk/javascript/client/state.js

```diff
@@ -18,8 +18,6 @@
 'use strict'
 
 const {Entry, TpStateGetRequest, TpStateGetResponse, TpStateSetRequest, TpStateSetResponse, Message} = require('../protobuf')
```

Comment:
This line is 126 characters. Probably not in this PR, but at some point we should refactor these files to enforce a character limit.